components/chip/chip.vue

```diff
@@ -203,10 +208,7 @@ export default {
 
   methods: {
     chipClasses () {
-      return [
-        this.$attrs['grouped-chip'] ? 'd-chip' : 'd-chip__label',
-        CHIP_SIZE_MODIFIERS[this.size],
-      ];
+      return [...this.labelClass, CHIP_SIZE_MODIFIERS[this.size]];
```

Comment:
Something broke with the styling here:

Chip
![Screenshot 2022-12-21 at 9 22 50 AM](https://user-images.githubusercontent.com/64808812/208966323-20ea972a-42fd-434d-8da1-9b2df61c309f.png)

Grouped Chip
![Screenshot 2022-12-21 at 9 23 00 AM](https://user-images.githubusercontent.com/64808812/208966346-f1a3f77d-90f1-4222-9bf3-bb491040da9c.png)